src/App/App.tsx

```diff
@@ -6,14 +6,14 @@ import Header from '../Header/Header';
 import GenresList from '../GenresList/GenresList';
 import PlaylistContainer from '../PlaylistContainer/PlaylistContainer';
 
-import { getPlaylist } from '../apiCalls';
+import { getPlaylist, getGenres } from '../apiCalls';
 
 import { CleanedAlbumTrack } from '../utils';
 
 interface IProps {}
 interface IState {
 	genres: [],
-	playlists: [CleanedAlbumTrack[]] | [],
+	playlists: CleanedAlbumTrack[][] | [],
```

Comment:
Interesting to know that this is the format for a nested array structure in Typescript.